docs/network.txt

```diff
@@ -3,37 +3,41 @@ IP Space
 
 Public IP space is 140.211.10.128/25, in a single VLAN shared with RTEMS.
 
-Buildbot is allocated 140.211.10.224/27 out of that /25.
+Buildbot is allocated 140.211.10.224/27 out of that /25:
+    140.211.10.225 -- 140.211.10.254
+
+(Misha: it looks like we are given the whole /25, so 140.211.10.224 can also be used.)
```

Comment:
Should the `/25` in the note be `/27`? If so, then the note is correct. The `/27` is a human splitting of the `/25`, so the netmask of the `/25` applies rather than what the `/27` would be. That means we can use `.224` but not `.255`.